metals/src/main/scala/scala/meta/internal/metals/MetalsLspService.scala

```diff
@@ -667,6 +667,7 @@ class MetalsLspService(
       userConfig,
     )
   )
+  buildClient.registerLogForwarder(debugProvider)
```

Comment:
Could it be a good idea to have some tests for the message forwarding? Not sure if it's worth it or even easily doable. :slightly_smiling_face:

Reply:
That's covered by the native test.